hail/python/hail/linalg/blockmatrix.py

```diff
@@ -1097,13 +1100,10 @@ def tofile(self, uri):
         --------
         :meth:`.to_numpy`
         """
-        n_entries = self.n_rows * self.n_cols
-        if n_entries >= 2 << 31:
```

Comment:
good catch, this should have been `1 << 31` which you fixed below

Reply:
Thanks!